src/routes/api/v1/organizations.js

```diff
@@ -1,16 +1,32 @@
 import express from 'express';
+import { celebrate } from 'celebrate';
 import { OrganizationController } from '../../../controllers';
 import {
-    asyncHandler,
-    checkOrganization
+  asyncHandler,
+  checkOrganization,
+  checkAuth,
 } from '../../../middlewares';
+import { organizationValidator } from './validators';
 
 const router = express.Router();
 
 router
-    .route('/').get(asyncHandler(OrganizationController.getAll));
+  .route('/')
+  .get(asyncHandler(OrganizationController.getAll))
+  .post(
+    checkAuth,
+    celebrate({ body: organizationValidator.post }),
+    asyncHandler(OrganizationController.post),
+  );
 
 router
-    .route('/:organizationId').get(checkOrganization, asyncHandler(OrganizationController.get));
+  .route('/:organizationId')
+  .all(asyncHandler(checkOrganization))
+  .get(asyncHandler(OrganizationController.get))
+  .put(
+    checkAuth,
+    celebrate({ body: organizationValidator.update }),
+    asyncHandler(OrganizationController.update),
+  );
 
```

Comment:
@oesukam Could we also have a delete feature for organizations?

Reply:
@gracelungu I can add it